build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 plugins {
-    java
+    `java-library`
```

Comment:
@bruno-garcia not sure if you have used `java` by purpose, but if not, as a library it is more efficient.